scripts/updateLPsAPR/index.ts

```diff
@@ -0,0 +1,136 @@
+import fs from 'fs'
+import { request, gql } from 'graphql-request'
+import BigNumber from 'bignumber.js'
+import { ChainId } from '@pancakeswap-libs/sdk'
+import chunk from 'lodash/chunk'
+import farmsConfig from '../../src/config/constants/farms'
+
+const BLOCK_SUBGRAPH = 'https://api.thegraph.com/subgraphs/name/pancakeswap/blocks'
+const STREAMING_FAST = 'https://bsc.streamingfast.io/subgraphs/name/pancakeswap/exchange-v2'
+
+const GET_BLOCK = (timestamp: string) => {
+  return gql`query getBlock {
+        blocks(first: 1, where: { timestamp_gt: ${timestamp}, timestamp_lt: ${timestamp + 600} }) {
+          number
+        }
+    }`
+}
+
+const FARMS_AT_BLOCK = (block: number | null, farms: string[]) => {
+  const blockString = block ? `block: {number: ${block}}` : ``
+  const addressesString = `["${farms.join('","')}"]`
+  return `pairs(
+      first: 30,
+      where: { id_in: ${addressesString} }
+      ${blockString}
+    ) {
+      id
+      volumeUSD
+      reserveUSD
+    }`
+}
```

Comment:
This is kind of dirty. I think we should use `graphql-request` with `gql`.

Reply:
This is mostly to keep stuff DRY, but I guess in this case as we don't request much I can just verbosely put everything into query below without this helper function 👍